buildSrc/src/main/groovy/com/uber/okbuck/rule/ExopackageAndroidLibraryRule.groovy

```diff
@@ -25,9 +35,11 @@ final class ExopackageAndroidLibraryRule extends BuckRule {
         printer.println("\tsource = '${mSourceCompatibility}',")
         printer.println("\ttarget = '${mTargetCompatibility}',")
         if (!mPostprocessClassesCommands.empty) {
+            String deps = "\$(JARS=(`find ${mGenDir} ! -name \"*-abi.jar\" ! -name \"*dex.dex.jar\" -name \"*.jar\"`); IFS=:; echo \"\${JARS[*]}\")"
+            String androidJar = mBootClasspath
             printer.println("\tpostprocess_classes_commands = [")
             mPostprocessClassesCommands.each { String command ->
-                printer.println("\t\t'${command}',")
+                printer.println("\t\t'export DEPS=${deps}; export ANDROID_JAR=${androidJar}; ${command}',")
```

Comment:
Lets try to extract this logic to a common place. There seems to be some non trivial amount of duplication here